tool/jt.rb

```diff
@@ -487,6 +489,13 @@ def build(*options)
       File.write(yytables, File.read(yytables).gsub('package org.jruby.parser;', 'package org.truffleruby.parser.parser;'))
     when 'options'
       sh 'tool/generate-options.rb'
+    when 'sulong'
```

Comment:
I thought we were going to just make Sulong a proper dependency. At that point, it would just build with our "mx build", wouldn't it?

Reply:
Yes it should be simpler after that.
But I'd rather still fix this until that is done (notably the `mx build --dependencies SULONG` part gets more complicated with a proper dependency).